docs/source/_static/html_snippets/chapters_content_directory_tree_only_yaml.html

```diff
@@ -1,11 +1,11 @@
 <div id="tree-diagram">
-	├── <span class="directory">algorithms</span><br>
-	│&nbsp;&nbsp;&nbsp;├── <span class="file">algorithms.yaml</span><br>
-	│&nbsp;&nbsp;&nbsp;└── <span class="directory">sections</span><br>
-	│&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;└── <span class="file">sections.yaml</span><br> class="file">lessons.yaml</span><br>
-	├── <span class="directory">artificial-intelligence</span><br>
-	│&nbsp;&nbsp;&nbsp;├── <span class="file">artificial-intelligence.yaml</span><br>
-	│&nbsp;&nbsp;&nbsp;└── <span class="directory">sections</span><br>
-	│&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;└── <span class="file">sections.yaml</span><br>
+	├── <span class="directory">algorithms/</span><br>
+	│   ├── <span class="directory">sections/</span><br>
+	│&nbsp;&nbsp;&nbsp;│&nbsp;&nbsp;&nbsp;└── <span class="file">sections.yaml/</span><br>
+	│   └── <span class="file">algorithms.yaml/</span><br>
+	├── <span class="directory">artificial-intelligence/</span><br>
+	│   ├── <span class="directory">sections/</span><br>
+	│&nbsp;&nbsp;&nbsp;│&nbsp;&nbsp;&nbsp;└── <span class="file">sections.yaml/</span><br>
+	│   └── <span class="file">artificial-intelligence.yaml/</span><br>
```

Comment:
Some of these probably shouldn't have `/` at the end since they're not directories